gdcmodels/force_merge_segements.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import argparse
+import threading
+import urllib3
+
+from itertools import cycle
+from time import sleep, time
+
+from elasticsearch import Elasticsearch
+from elasticsearch.exceptions import ConnectionTimeout
+
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+
+def get_parser():
```

Comment:
also, consider separating parser logic from the "service" logic

Reply:
Sorry, I do not get it. Could you please explain a little bit more?